handlers/cars.go

```diff
@@ -1,11 +1,11 @@
 package handlers
 
 import (
-	"log"
 	"net/http"
-
+    "errors"
 	"github.com/gorilla/mux"
 	"github.com/sikozonpc/fullstackgo/components"
+	"github.com/sikozonpc/fullstackgo/store"
```

Comment:
I would just format the errors package import
```suggestion
        "errors"
	"github.com/gorilla/mux"
	"github.com/sikozonpc/fullstackgo/components"
	"github.com/sikozonpc/fullstackgo/store"
```